components/Modal/CreateImageUpload.js

```diff
@@ -311,23 +320,70 @@ class CreateImageUploadModal extends React.Component {
       uploadService
     } = this.state;
 
-    const providerCheckbox = (provider, displayName) => (
-      <FormGroup
-        label={this.props.intl.formatMessage({ id: `${provider}-form`, defaultMessage: "Upload image" })}
-        fieldId="provider-checkbox"
-      >
-        <Checkbox
-          value={provider}
-          isChecked={this.state.uploadService === provider}
-          onChange={this.handleUploadService}
-          label={this.props.intl.formatMessage({
-            id: `${provider}-checkbox`,
-            defaultMessage: `Upload to ${displayName}`
-          })}
-          id={`${provider}-checkbox`}
-          aria-labelledby="provider-checkbox"
-        />
-      </FormGroup>
+    const awsProviderCheckbox = (
+      <div className="pf-c-form__group">
+        <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+          <span className="pf-l-flex__item pf-c-form__label-text">
+            <FormattedMessage defaultMessage="Upload image" />
+          </span>
+          <Popover
+            id="popover-help"
+            bodyContent={
+              <TextContent>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="
+                      Image Builder can upload images you create to an {bucket} in AWS and then import them into EC2. When the image build is complete
```

Comment:
I didn't catch this in my first review. I'm wondering if the first time we mention AWS, we should use this:
> Amazon Web Services (AWS)

And then after that, just use "AWS".